config/src/main/java/org/hyperledger/besu/config/IbftConfigOptions.java

```diff
@@ -75,6 +77,14 @@ public int getFutureMessagesMaxDistance() {
         ibftConfigRoot, "futuremessagesmaxdistance", DEFAULT_FUTURE_MESSAGES_MAX_DISTANCE);
   }
 
+  public Optional<String> getMiningBeneficiary() {
+    return JsonUtil.getString(ibftConfigRoot, "miningbeneficiary");
+  }
+
+  public long getBlockRewardEth() {
+    return JsonUtil.getLong(ibftConfigRoot, "blockrewardeth", DEFAULT_IBFT_BLOCK_REWARD_ETH);
```

Comment:
Don't think this need eth as a suffix, no other fields do that. Should be clear enough that it's eth.

Reply:
done